tests/show_results.py

```diff
@@ -52,6 +54,29 @@ def translate_path(self, path: str) -> str:
             path += "/"
         return path
 
+    def do_POST(self) -> None:
+        if self.path == "/fixtures.json" and os.path.exists(FIXTURES_PATH):
```

Comment:
As `FIXTURES_PATH` is already a `Path` (not a string), it is possible to use `FIXTURES_PATH.exists()`

Reply:
fixed https://github.com/trezor/trezor-firmware/commit/a2900711241302cb3b2b7f4d1cac0f538ecf4960